core/app/c/[communitySlug]/pubs/create/page.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Metadata } from "next";
+
+import { notFound } from "next/navigation";
+
+import type { CommunitiesId, PubsId, StagesId } from "db/public";
+import { Capabilities } from "db/src/public/Capabilities";
+import { MembershipType } from "db/src/public/MembershipType";
+import { Button } from "ui/button";
+
+import { ContentLayout } from "~/app/c/[communitySlug]/ContentLayout";
+import { PageTitleWithStatus } from "~/app/components/pubs/PubEditor/PageTitleWithStatus";
+import { PubEditor } from "~/app/components/pubs/PubEditor/PubEditor";
+import { getPageLoginData } from "~/lib/authentication/loginData";
+import { userCan } from "~/lib/authorization/capabilities";
+import { getCommunityBySlug } from "~/lib/db/queries";
+
+export async function generateMetadata({
+	params: { communitySlug },
+}: {
+	params: { pubId: string; communitySlug: string };
+}): Promise<Metadata> {
+	const community = await getCommunityBySlug(communitySlug);
+	if (!community) {
+		return { title: "Community Not Found" };
+	}
+
+	return { title: `Create pub in ${community.name}` };
+}
+
+export default async function Page({
+	params,
+	searchParams,
+}: {
+	params: { communitySlug: string };
+	searchParams: Record<string, string>;
+}) {
+	const { communitySlug } = params;
+
+	const { user } = await getPageLoginData();
+
+	const community = await getCommunityBySlug(communitySlug);
+
+	if (community === null) {
+		notFound();
+	}
+
+	const communityId = community.id as CommunitiesId;
+
+	const canCreatePub = await userCan(
+		Capabilities.createPub,
```

Comment:
this was originally on `MembershipType.pub` but I moved it to `MembershipType.community` which I think makes more sense, but let me know if that's not right!

Reply:
That's right, definitely a mistake! I think it's both a community and stage capability.

Reply:
I guess the stage part of that is a bit of an issue if the stage doesn't get selected until they're on the create page...

Reply:
hmm maybe we filter the stage dropdown by only the ones the user has capabilities for?